static/creation/templates/template2.html

```diff
@@ -0,0 +1,200 @@
+<!-- Contributor -->
```

Comment:
How's `template2.html` different from `modals.html`? I think one of these need to be removed.